rpm/libcontentaction-qt5.spec

```diff
@@ -17,6 +17,8 @@ BuildRequires:  pkgconfig(Qt5SystemInfo)
 BuildRequires:  pkgconfig(Qt5Qml)
 BuildRequires:  python3-base
 BuildRequires:  qt5-qttools-linguist
+# TODO: Add version, needs recent enough invoker
+Requires: mapplauncherd
```

Comment:
I think you could already add next version here.

Reply:
Added.